packages/terminal-external/src/electron-browser/terminal-external-preference.ts

```diff
@@ -0,0 +1,66 @@
+/********************************************************************************
+ * Copyright (C) 2021 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { interfaces } from 'inversify';
+import {
+    createPreferenceProxy,
+    PreferenceProxy,
+    PreferenceSchema,
+    PreferenceService,
+    PreferenceContribution
+} from '@theia/core/lib/browser';
+import { TerminalExternalConfiguration } from '../common/terminal-external';
+
+// The default terminal paths to be displayed on preferences.
```

Comment:
The documentation is misleading, they are not paths and are not only for display purposes.

We can update to something along the lines of:

```suggestion
// The default terminal exec preferences for each operating system.
```